nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
```

Comment:
- [ ] Following the project requirements, you should create TrimmerDecorator class that inherits from the base Decorator class:
- In TrimmerDecorator class, implement a method correct_name that makes sure that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.